tlslite/keyexchange.py

```diff
@@ -223,17 +223,17 @@ def makeClientKeyExchange(self):
         clientKeyExchange.createRSA(self.encPremasterSecret)
         return clientKeyExchange
 
-# the DHE_RSA part comes from IETF ciphersuite names, we want to keep it
-#pylint: disable = invalid-name
-class DHE_RSAKeyExchange(KeyExchange):
+
+# FFDHE without signing serverKeyExchange useful for anonymous DH
```

Comment:
comments starting with '# ' are just that, comments, comments inside """ marks are documentation strings that epydoc uses to create documentation

this comment can be incorporated into the `ADHKeyExchange` documentation comment